pkg/whiteflag/white_flag.go

```diff
@@ -178,8 +181,29 @@ func ComputeWhiteFlagMutations(s *storage.Storage, msIndex milestone.Index, cach
 			inputOutputs = append(inputOutputs, output)
 		}
 
+		// Dust validation
+		dustValidation := iotago.NewDustSemanticValidation(iotago.DustAllowanceDivisor, func(addr iotago.Serializable) (dustAllowanceSum uint64, amountDustOutputs int64, err error) {
+			address := addr.(iotago.Address)
+			dustAllowanceBalance, dustOutputCount, err := s.UTXO().ReadDustForAddress(address)
+			if err != nil {
+				return 0, 0, err
+			}
+
+			dustDiff, found := wfConf.dustDiff[address.String()]
```

Comment:
you should convert the address bytes to string, but the String function uses hex encoding